src/cluster_legacy.c

```diff
@@ -3252,6 +3255,7 @@ int clusterIsValidPacket(clusterLink *link) {
                 explen += extlen;
                 ext = getNextPingExt(ext);
             }
+            link->support_extension = 1;
```

Comment:
Nit:

Btw if we move the extensions processing ahead of the core message processing. I would prefer this check to be moved to `clusterProcessPacket`, this feels like a bit of bypassing.

Reply:
Tbh, I don't like the latest commit, most of the flags are unrelated to link and we are starting off by the 10th bit. If you want to save memory, we could start with another macro for links `LINK_EXTENSIONS_SUPPORTED (1<<0)` or use the `ClientFlags` approach. Will leave it upto you.

Reply:
Sorry if this does not suit you well. I just thought maybe we can re-use the node flags, but i do agree it is kind of overdoing.